Gopkg.lock

```diff
@@ -61,12 +61,12 @@
   revision = "6480d4af844c189cf5dd913db24ddd339d3a4f85"
 
 [[projects]]
-  branch = "master"
-  digest = "1:08188cf7ce7027b22e88cc23da27f17349a0ba7746271a60cbe0a70266c2346f"
+  digest = "1:2cd7915ab26ede7d95b8749e6b1f933f1c6d5398030684e6505940a10f31cfda"
   name = "github.com/ghodss/yaml"
   packages = ["."]
   pruneopts = "UT"
-  revision = "25d852aebe32c875e9c044af3eef9c7dc6bc777f"
+  revision = "0ca9ea5df5451ffdf184b4428c902747c2c11cd7"
+  version = "v1.0.0"
```

Comment:
This actually goes back from latest commit to the only 1.0.0 release from 2017. Not sure is this desired

Reply:
Me neither. I only ran `dep ensure -update`.